server/pension/models.py

```diff
@@ -21,13 +21,13 @@ class InstrumentType(object):
     TYPE2 = 2
 
     choices = (
-        (TYPE1, _('')),
-        (TYPE2, _('')),
+        (TYPE1, _('ניירות ערך סחירים')),
```

Comment:
הערך המתקבל מהטבלה הוא בעברית. אם נשים ערכים באנגלית זה לא יעבוד. 

Sent from my iPhone

> On 17 May 2016, at 19:55, Nir Galon <notifications@github.com> wrote:
> 
> In server/pension/models.py:
> 
> > @@ -21,13 +21,13 @@ class InstrumentType(object):
> >      TYPE2 = 2
> >  
> >      choices = (
> > -        (TYPE1, _('')),
> > -        (TYPE2, _('')),
> > +        (TYPE1, _('ניירות ערך סחירים')),
> @ordavidil @Lemore "ניירות ערך סחירים" is "Marketable Security" (or "Securities" in plurals). There is another option of "ניירות ערך לא סחירים" and that's "Non-Marketable Security".
> Those are under the column "instrument_type".
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>